pymc3/tests/test_distributions.py

```diff
@@ -415,7 +425,7 @@ def test_wald(self, value, mu, lam, phi, alpha, logp):
         with Model() as model:
             Wald('wald', mu=mu, lam=lam, phi=phi, alpha=alpha, transform=None)
         pt = {'wald': value}
-        assert_almost_equal(model.fastlogp(pt), logp, decimal=6, err_msg=str(pt))
+        assert_almost_equal(model.fastlogp(pt), logp, decimal=select_decimal(float64=6, float32=1), err_msg=str(pt))
```

Comment:
Yes IIRC.  I think this was returning a logp whose value was ~1E5, so 6 decimal places is actually 11 digits of precision---totally unattainable on float32.